web/web.go

```diff
@@ -224,9 +225,11 @@ func (h *Handler) Quit() <-chan struct{} {
 	return h.quitCh
 }
 
-// Reload returns the receive-only channel that signals configuration reload requests.
-func (h *Handler) Reload() <-chan struct{} {
-	return h.reloadCh
+// Reload returns two cahnnels:
+//   - receive-only channel that signals configuration reload requests.
+//   - send-only channel with errors happened during configuration reload.
```

Comment:
@brian-brazil I've fixed logging stuff, any ideas how this interface can be improved?